lib/rubocop/remote_config.rb

```diff
@@ -32,6 +32,8 @@ def file
     end
 
     def inherit_from_remote(file, path)
+      file.gsub!(%r{^\./(.*)$}) { Regexp.last_match(1) } if file.start_with?('./')
```

Comment:
It looks like `delete_prefix` can be used. Also, the argument may be unexpected to have side effect by bang method. Can you replace it?

```suggestion
      file = file.delete_prefix('./')
```

Reply:
I didn't know there was such a useful method! Thank you!
I'll fix it.

Reply:
Fixed in 60e6f47 😃